src-django/authentication/models.py

```diff
@@ -1 +1,32 @@
-from signals import user_post_save  # noqa
+from django.db import models
+from django.contrib.auth.models import User
+import datetime
+import hashlib
+import random
+
+EMAIL_CONFIRMATION_KEY_TTL_DAYS = 2
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(User, related_name="profile", on_delete=models.CASCADE)
+    is_email_confirmed = models.BooleanField(default=False)
+
+    class Meta:
+        app_label = 'authentication'
+
+
+class EmailConfirmationKey(models.Model):
+    user = models.OneToOneField(User)
+    key = models.CharField(max_length=40, null=False, blank=False)
```

Comment:
You don't need to explicitly say that these aren't false or blank. It is by default.